play/src/front/Phaser/Game/GameScene.ts

```diff
@@ -486,13 +488,12 @@ export class GameScene extends DirtyScene {
         });
 
         this.load.scenePlugin("AnimatedTiles", AnimatedTiles, "animatedTiles", "animatedTiles");
-
         if (this.wamUrlFile) {
             const absoluteWamFileUrl = new URL(this.wamUrlFile, window.location.href).toString();
 
             this.superLoad.loadPromise(
                 axiosWithRetry.get(absoluteWamFileUrl).then((response) => {
-                    const wamFileResult = WAMFileFormat.safeParse(response.data);
+                    const wamFileResult = WAMFileFormat.safeParse(wamFileMigration.migrate(response.data));
```

Comment:
Here we should try catch and handle the error message just like when wamFileResult.success === false
(remove loader, set errorScreenStore...)